src/pages/index.js

```diff
@@ -75,8 +168,33 @@ function handleCardPreview(cardData) {
 
 // Event Handlers
 
-function handleProfileEditSubmit({ name, description }) {
-  userInfo.setUserInfo({ name, description });
+function handleProfileEditSubmit(userData) {
+  const { name, description } = userData;
+  api.updateUserProfile({ name, about: description }).then((updateUserData) => {
+    console.log(updateUserData);
+    userInfo.setUserInfo({
+      name: updateUserData.name,
+      description: updateUserData.about,
+    });
+
+    editProfilePopup.close();
+  });
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**